TradingOnTalentURAP/matching.py

```diff
@@ -0,0 +1,567 @@
+import os
+import sys
+import glob
+import json
+import re
+import csv
+import multiprocessing
+
+locations = pd.read_csv('university_czone.csv')
+universities = pd.read_csv('universities.csv',header=None)
+universities.columns = ['name','student_count']
+
+# Read in reference schools
+
+schools_f = open('herd_2008_unique_school.csv','rt',encoding='utf-8')
+schools_c = csv.reader(schools_f)
+
+schools_d = []
+schools = []
+for school in schools_c:
+    schools_d.append( (set([i for i in 
+                            school[9].lower().translate({ord(c): ' ' 
+                            for c in '|[]_*?!@#$,.-;:"/()'}).split() 
+                            if i not in ['in','at','of','for','studies','all',
+                                         'campuses','main','campus','and','the',
+                                         '&']]), school[9]))
+
+schools = [i[0] for i in schools_d]
+schools = schools[1:]
+schools_d = schools_d[1:]
+
+# Colloquial names found on Wikipedia
+
+colloquial_names = {
```

Comment:
This is awesome, but, I would still like these to be in some csv file that gets read in at the start--to allow future modifications without changing the code...